modules/mirror-registry-release-notes.adoc

```diff
@@ -142,7 +142,7 @@ IPv6 is currently unsupported on _mirror registry for Red Hat OpenShift_ remote
 
 * A new feature flag, `--quayStorage`, has been added. With this flag, users with root privileges can manually set the location of their Quay persistent storage.
 
-* A new feature flag, `--pgStorage`, has been added. With this flag, users with root privileges can manually set the location of their Postgres persistent storage.
+* A new feature flag, `--pgStorage`, has been added. By specifying this flag, you can manually set the location for the Postgres persistent storage.
```

Comment:
I would suggest also fixing the working for the `quayStorage` line as well or removing this change. It seems a bit inconsistent to change one and not the other.

Reply:
Thanks, @barbacbd . I updated the Quay line also. Would you be OK to add a LGTM?